docs/developer/decision-records/2022-06-13-registration-service-api/README.md

```diff
@@ -0,0 +1,49 @@
+# Registration Service API
+
+Registration Service is a component of a Dataspace Authority. 
+One of the responsibilities of the Registration Service is the management of the participants in the Dataspace.
+
+The Registration Service exposes an API that offers the following operations:
+- Add a participant to the Dataspace
+- List participants in the Dataspace
+
+The Registration Service is written in Java and uses the runtime framework and modules from [EDC](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector).
+
+## OpenApi 
+
+The Registration Service API definition is available in an [OpenApi yaml file](../../../../resources/openapi/yaml/registration-service.yaml). 
+
+## API Rest client
+
+The Registration Service API can be called using the [rest-client](../../../../rest-client) generated from OpenApi definition.
```

Comment:
Not sure we really want to include the Java client code in the "server" code. If anything, we should publish this as separate Maven artifact down the line.

Reply:
It's mentioned below that the Registration Service is currently not published to any artifactory so the client can be used for local development. Would you change/add something to that? Or are you suggesting to remove `API Rest Client` section from the ADR?

Reply:
I'm doubtful that there is much value in shipping REST client code directly with the server code. 
It's OK to leave it in for now, for development/testing purposes, but eventually the client code should not be shipped with the server code.